data/pilots/first-order/tie-se-bomber.json

```diff
@@ -0,0 +1,159 @@
+      "type": "Reload"
+    }
+  ],
+  "pilots": [
+    {
+      "name": "Breach",
+      "xws": "breach",
+      "initiative": 5,
+      "limited": 1,
+      "shipAbility": {
+        "name": "Pursuit Thrusters",
+        "text": "During the System Phase, you may perform a [Boost] action."
+      },
+      "hyperspace": true,
+      "cost": 0,
+      "slots": [
+        "Talent",
+        "Tech",
+        "Torpedo",
+        "Missile",
+        "Gunner",
+        "Device",
+        "Modification"
+      ],
+      "ability": "After you fully execute a maneuver or perform a [Boost] action, if you moved through an enemy ship, you may acquire a lock on that ship.",
+      "ffg": -999,
+      "artwork": "???",
+      "image": "???",
+      "caption": "Ordnance Expert",
+      "keywords": ["TIE"]
+    },
+    {
+      "name": "Scorch",
+      "xws": "scorch-tiesebomber",
+      "initiative": 4,
+      "limited": 1,
+      "shipAbility": {
+        "name": "Pursuit Thrusters",
+        "text": "During the System Phase, you may perform a [Boost] action."
+      },
+      "hyperspace": true,
+      "cost": 0,
+      "slots": [
+        "Talent",
+        "Tech",
+        "Torpedo",
+        "Missile",
+        "Gunner",
+        "Device",
+        "Modification"
+      ],
+      "ability": "While a friendly ship at range 0-1 performs a primary attack, it may spend 1 [Hit] result. If it does, after defending, the defender gains 1 strain token.",
+      "ffg": -999,
+      "artwork": "???",
+      "image": "???",
+      "caption": "Jad Bean",
+      "keywords": ["TIE"]
+    },
+    {
+      "name": "Grudge",
+      "xws": "grudge",
+      "initiative": 2,
+      "limited": 1,
+      "shipAbility": {
+        "name": "Pursuit Thrusters",
+        "text": "During the System Phase, you may perform a [Boost] action."
+      },
+      "hyperspace": true,
+      "cost": 0,
+      "slots": [
+        "Talent",
+        "Tech",
+        "Torpedo",
+        "Missile",
+        "Gunner",
+        "Device",
+        "Modification"
+      ],
+      "ability": "While a friendly bomb or mine at range 0-2 detonates, each time attack dice are rolled to resolve its effect, you may reroll up to 1 of those dice.",
+      "ffg": -999,
+      "artwork": "???",
+      "image": "???",
+      "caption": "Hateful Harrier",
+      "keywords": ["TIE"]
+    },
+    {
+      "name": "First Order Cadet",
+      "xws": "firstordercadet",
+      "initiative": 3,
+      "limited": 0,
+      "shipAbility": {
+        "name": "Pursuit Thrusters",
+        "text": "During the System Phase, you may perform a [Boost] action."
+      },
+      "hyperspace": true,
+      "cost": 0,
+      "slots": [
+        "Talent",
+        "Tech",
+        "Torpedo",
+        "Missile",
+        "Gunner",
+        "Device",
+        "Modification"
+      ],
+      "text": "The live-fire exercices of the First Order are intended to root out the weak, enforcing the maxim that only the strong survive.",
```

Comment:
Text should be: "The live-fire exercises of the First Order are intended to root out the weak, enforcing the maxim that only the strong survive."